src/libappimage/desktop_integration/integrator/Integrator.cpp

```diff
@@ -177,7 +201,7 @@ namespace appimage {
                         try {
                             Logger::warning("Using .DirIcon as default app icon");
                             auto dirIconData = resourcesExtractor.extract(dirIconPath);
-                            deployApplicationIcon(desktopEntryIconName, dirIconData);;
+                            deployIcon("apps", desktopEntryIconName, dirIconData);;
```

Comment:
Don't understand what this change implies. Why is there no comments at all in this file btw?

Reply:
This change makes the function accept an additional parameter (the icon group) making it more generic.

Also there are comments in the file, please take a better look.

Reply:
There's not a single comment around this code block. There's like 5 comments for a file with > 300 lines of code. That ratio is kind of bad. That's what I meant.

Reply:
There is one a little bit above
https://github.com/AppImage/libappimage/blob/7ef3c6198c484f2435ff0576e73a9fdc4df7a11e/src/libappimage/desktop_integration/integrator/Integrator.cpp#L198-L204

Anyway I will review the file and add comments were required

Reply:
I have added a few more comments. But I guess that there much more that 5 comments in the file, please see the whole file not only the committed changes. Also consider that the methods that actually do something are documented on their headers.